test/Interop/Cxx/operators/non-member-inline.swift

```diff
@@ -169,4 +169,20 @@ OperatorsTestSuite.test("pipe pipe (||)") {
   expectEqual(true, result.value)
 }
 
+OperatorsTestSuite.test("plus plus (++)") {
+  let lhs = LoadableIntWrapper(value: 42)
+
+  let result = lhs += 1
```

Comment:
These tests look like they're failing. The rhs needs to be a `LoadableIntWrapper` not an integer.